package/gluon-core/luasrc/lib/gluon/upgrade/820-dns-config

```diff
@@ -25,22 +25,33 @@ if dns.servers then
 	})
 end
 
+
+local function set_dns_record(name, host, sectionname)
+		uci:section('dhcp', 'domain', sectionname, {
+			name = name,
+			ip = host,
+		})
+end
+
+-- remove nextnode4_* and nextnode6_*
+uci:foreach("dhcp", "domain", function(s)
+	for key, value in pairs(s) do
+		if (key == ".name" and string.gmatch(value, "nextnode[46].*")) then
+			uci:delete("dhcp", value)
+		end
+	end
```

Comment:
```lua
if string.match(s['.name'], '^nextnode[46]') then
```
- No need to loop over the pairs of *s*
- `^` anchor was missing (and `.*` was redundant)
- Use *match* instead of *gmatch*, was we're only interested in the first match

The comment does not really match the code: the code doesn't care about the underscore, the comment should be adjusted.

Reply:
done